model_cards/Helsinki-NLP/opus-mt-cmn+cn+yue+ze_zh+zh_cn+zh_CN+zh_HK+zh_tw+zh_TW+zh_yue+zhs+zht+zh-fi/README.md

```diff
@@ -0,0 +1,15 @@
+opus-2020-01-20.zip
+
+* dataset: opus
+* model: transformer-align
+* pre-processing: normalization + SentencePiece
+* download: [opus-2020-01-20.zip](https://object.pouta.csc.fi/OPUS-MT-models/cmn+cn+yue+ze_zh+zh_cn+zh_CN+zh_HK+zh_tw+zh_TW+zh_yue+zhs+zht+zh-fi/opus-2020-01-20.zip)
```

Comment:
Do you think you could automatically add the language tags as well? cc @julien-c 

Also, at some point, we will need to explain how the model name are made. In particular, since the model cards don't comprise explicit long-form names of the languages on which the models are trained.

Reply:
(i.e. at some point we'll have to think about the user and how he could find a model he would like to use in this 1001 models)

Reply:
Yeah I agree.
I asked @jorgtied how he thought we should handle that.
here it looks like the `zh_` prefix represents the chinese family, but also it's own language. Maybe these are actually dataset codes?

I could add:
```python
src: [cmn, cn, yue, ze_zh, zh_cn, zh_CN, zh_HK, zh_tw, zh_TW, zh_yue, zhs, zht, zh]
tgt: fi
```
to the markdown, or attach some metadata to the config.

Eventually, we could also try to aggregate the eval statistics and make a little table that allows a user to figure out: _given documents in language x, which models can I use (easy) and which model should I use (harder)._